larpandoracontent/LArHelpers/LArRandomHelper.cc

```diff
@@ -0,0 +1,21 @@
+/**
+ *  @file   larpandoracontent/LArHelpers/LArRandomHelper.cc
+ *
+ *  @brief  Implementation of the random helper class.
+ *
+ *  $Log: $
+ */
+
+#include <random>
+
+namespace lar_content
+{
+
+int GetIntsInRange(const int low, const int high, std::mt19937 &eng)
```

Comment:
Header for this should explain why I had to write my own, rather than use the built in.